SteamReview/SteamReview/Steam.cpp

```diff
@@ -213,7 +233,7 @@ bool FSteam::AddNewGameToCategory(const FGame InGame)
 
```

Comment:
Comment for line 232: Same as before, 'follow' mightn't be the word you're looking for. Try switching it to continue and you're good to go